src/core/src/core_logic/PatchInstaller.py

```diff
@@ -261,20 +287,43 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
                     self.last_still_needed_packages.pop(index)
                     self.last_still_needed_package_versions.pop(index)
                     installed_update_count += 1
+                    number_of_dependencies_installed += 1
                 else:
                     # status is not logged by design here, in case you were wondering if that's a bug
                     message = " - [Info] Dependency appears to have failed to install (note: it *may* be retried): " + str(dependency) + "(" + str(dependency_version) + ")"
                     self.composite_logger.log_debug(message)
+                    number_of_dependencies_failed += 1
 
             # dependency package result management fallback (not reliable enough to be used as primary, and will be removed; remember to retain last_still_needed refresh when you do that)
-            installed_update_count += self.perform_status_reconciliation_conditionally(package_manager, condition=(attempted_parent_update_count % Constants.PACKAGE_STATUS_REFRESH_RATE_IN_SECONDS == 0))  # reconcile status after every 10 attempted installs
+            installed_update_count += self.perform_status_reconciliation_conditionally(package_manager, condition=(self.attempted_parent_update_count % Constants.PACKAGE_STATUS_REFRESH_RATE_IN_SECONDS == 0))  # reconcile status after every 10 attempted installs
+
+            patch_install_perf_log = {"Package Name": package, "Package version": version, "Package and dependencies": str(package_and_dependencies), "Package and dependency versions": str(package_and_dependency_versions),
+                                      "Package install result": str(install_result), "Number of dependencies installed": str(number_of_dependencies_installed), "Number of dependencies failed to install": str(number_of_dependencies_failed)}
```

Comment:
nit: inconsistent spacing

Reply:
I am not able to find the inconsistency here. Could you please point out specifically where is inconsistency?
Might be the view in diff is confusing. Could you please go to "View file" and check if there is inconsistency?